src/control.class.js

```diff
@@ -217,6 +249,59 @@
       return point;
     },
 
+    /**
+     * Returns the coords for this control based on object values.
+     * @param {Number} objectAngle angle from the fabric object holding the control
+     * @param {Number} objectCornerSize cornerSize from the fabric object holding the control (or touchCornerSize if
+     *   isTouch is true)
+     * @param {Number} centerX x coordinate where the control center should be
+     * @param {Number} centerY y coordinate where the control center should be
+     * @param {boolean} isTouch true if touch corner, false if normal corner
+     */
+    calcCornerCoords: function(objectAngle, objectCornerSize, centerX, centerY, isTouch) {
+      var controlTriangleAngle,
+          cornerHypotenuse,
+          xSize = (isTouch) ? this.touchCornerSizeX : this.cornerSizeX,
+          ySize = (isTouch) ? this.touchCornerSizeY : this.cornerSizeY;
+      if (xSize && ySize) {
+        // handle custom corner sizes
+        controlTriangleAngle = fabric.util.radiansToDegrees(Math.atan2(ySize, xSize));
+        cornerHypotenuse = Math.sqrt(xSize * xSize + ySize * ySize) / 2;
+      }
+      else {
+        // use default object corner sizes
+        controlTriangleAngle = 45;
+        /* 0.7071067812 stands for sqrt(2)/2 */
+        cornerHypotenuse = objectCornerSize * 0.7071067812;
+      }
+      var newTheta = controlTriangleAngle - fabric.util.degreesToRadians(objectAngle),
```

Comment:
Here we are subtracting degrees with radians

Reply:
Thanks, I fixed this in my test code but didn't transfer it late last night when I committed. I need to not let lack of sleep cause blunders.